test.js

```diff
@@ -1,5 +1,6 @@
 import test from 'ava';
-import figures, {replaceSymbols, mainSymbols, windowsSymbols} from '.';
+// eslint-disable-next-line unicorn/import-index, import/extensions
```

Comment:
Removing this comment requires upgrading `xo`.
However, upgrading `xo` depends on #79 (dropping support for Node <12).